src/mpyl/utilities/docker/__init__.py

```diff
@@ -303,11 +305,23 @@ def build(
 
 def login(logger: Logger, registry_config: DockerRegistryConfig) -> None:
     logger.info(f"Logging in with user '{registry_config.user_name}'")
-    docker.login(
-        server=f"https://{registry_config.host_name}",
-        username=registry_config.user_name,
-        password=registry_config.password,
-    )
+    if registry_config.provider == "azure":
+        docker.login(
+            server=f"https://{registry_config.host_name}",
+            username=registry_config.user_name,
+            password=registry_config.password,
+        )
+    if registry_config.provider == "aws":
+        docker.login_ecr(
+            aws_access_key_id=registry_config.user_name,
+            aws_secret_access_key=registry_config.password,
+            region_name="eu-central-1",
```

Comment:
Why is this hardcoded instead of configured?

Reply:
removed it and added it as an optional string to the config file